lib/hanami/action/application_action.rb

```diff
@@ -70,6 +70,14 @@ module InstanceMethods
         attr_reader :view
         attr_reader :view_context
 
+        protected
+
+        def handle(request, response)
+          if respond_to?(:view) && view
```

Comment:
@timriley why does it check `respond_to?(:view)`? I see `attr_reader :view` on line 71 so...every view responds to this method, no?

Reply:
@solnic you're right, this wasn't needed, thanks for checking! Removing it now.